src/graph/GoExecutor.cpp

```diff
@@ -452,6 +447,135 @@ void GoExecutor::onStepOutResponse(RpcResponse &&rpcResp) {
 }
 
 
+void GoExecutor::maybeFinishExecution(RpcResponse &&rpcResp) {
+    auto requireDstProps = expCtx_->hasDstTagProp();
+    auto requireEdgeProps = !expCtx_->aliasProps().empty();
+
+    // Non-reversely traversal, no properties required on destination nodes
+    // Or, Reversely traversal but no properties on edge and destination nodes required.
+    if ((!requireDstProps && !isReversely()) ||
+            (isReversely() && !requireDstProps && !requireEdgeProps)) {
```

Comment:
For reversely traversal,   requireDstProps  maybe renamed as requireSrcProps. 
The variable name here makes me confused. 

PS.  we should add TODO here,  because the in-edges props will be stored with the dst Id in the future.